app/src/pages/playground/InvocationParametersForm.tsx

```diff
@@ -16,113 +30,221 @@ const FormField = ({
   value,
   onChange,
 }: {
-  field: keyof InvocationParametersSchema;
-  value: InvocationParametersSchema[keyof InvocationParametersSchema];
-  onChange: (
-    value: InvocationParametersSchema[keyof InvocationParametersSchema]
-  ) => void;
+  field: InvocationParameter;
```

Comment:
might consider renaming this component to something like `InvocationParameterFormField`, i know it's in this file and not exported but I think i prefer the explicitness, caught me off guard a bit when i saw it down below